docs/schemas/core/IfcKernel/Entities/IfcProduct.md

```diff
@@ -32,10 +32,10 @@ Reference to the representations of the product, being either a representation (
 Reference to the _IfcRelAssignsToProduct_ relationship, by which other products, processes, controls, resources or actors (as subtypes of _IfcObjectDefinition_) can be related to this product.
 
 ### PositionedRelativeTo
-
+Reference to the _IfcRelPositions_ relationship, which defines its position relative to a positioning element.
```

Comment:
> Reference to the _IfcRelPositions_ relationship, which defines its position relative to a positioning element.

"its" is ambiguous -> "defines the position of the product"

But we also had a comment regarding this. That we need to more carefully distinguish it from placement -> "describe information on positioning dependencies on a semantic layer"